Resources/public/mapbender.element.digitizer.js

```diff
@@ -457,7 +457,71 @@
 
                         // http://dev.openlayers.org/docs/files/OpenLayers/Control-js.html#OpenLayers.Control.events
                         controlEvents: {
-                            featureadded: function(event) {
+
+                            /**
+                             * This function allows the easy use of the olEvent onStart( called on drag start) with the yml configurration
+                             * e.G. to prevent the move or add additional data on move
+                             * */
+                            onStart:               function(feature, px) {
+
+                                var control = this;
+                                var schema = feature.schema;
+                                var attributes = feature.attributes;
+                                var preventDefault = false;
+
+                                if(!schema.hooks || !schema.hooks.onStart) {
+                                    return;
+                                }
+
+                                try {
+                                    preventDefault = eval(schema.hooks.onStart);
+                                } catch (e) {
+
+                                    $.notify(e);
+                                    return;
+                                }
+
+                                if(preventDefault) {
+                                    $.notify(translate('move.denied'));
+                                    control.cancel();
+
+                                }
+                            },
+
+                            /**
+                             * This function allows the easy use of the olEvent onModificationStart with the yml configurration
+                             * e.G. to prevent the modification or add additional data on modification
+                             * */
+                            onModificationStart:   function event(feature) {
+
+                                var control = this;
+                                var schema = feature.schema;
+                                var attributes = feature.attributes;
+                                var preventDefault = false;
+
+                                if(!schema.hooks || !schema.hooks.onModificationStart) {
+                                    return;
+                                }
+
+                                try {
+                                    preventDefault = eval(schema.hooks.onModificationStart);
+                                } catch (e) {
+
+                                    $.notify(e);
+                                    return;
+                                }
+
+                                if(preventDefault) {
```

Comment:
I don't really understand what this method is doing... If `preventDefault` is `null`, `false` or `undefined` it will do nothing, and when it prevents something it will reactivate the plugin? Why?